debian/changelog

```diff
@@ -1,9 +1,12 @@
-cloud-init (23.4-0ubuntu1~22.04.2) UNRELEASED; urgency=medium
+cloud-init (23.4.1-0ubuntu1~22.04.1) jammy; urgency=medium
 
   * d/p/retain-apt-pre-deb822.patch:
     - Disable apt source list generation with DEB822 style
+  * Upstream snapshot based on 23.4.1. (LP: #2045582).
```

Comment:
changeset, version and pkg build look good. We are missing capture of the refreshed patches in debian/changelog. We also missed this in our first -proposed upload of 23.4-0.

Here's a suggested diff for jammy
```diff
--- a/debian/changelog
+++ b/debian/changelog
@@ -2,6 +2,9 @@ cloud-init (23.4.1-0ubuntu0~22.04.1) jammy; urgency=medium
 
   * d/p/retain-apt-pre-deb822.patch:
     - Disable apt source list generation with DEB822 style
+  * refresh patches:
+    - d/p/status-do-not-remove-duplicated-data.patch
+  * d/changelog: amend 23.4-0 refresh patches entry
   * Upstream snapshot based on 23.4.1. (LP: #2045582).
     List of changes from upstream can be found at
     https://raw.githubusercontent.com/canonical/cloud-init/23.4.1/ChangeLog
@@ -15,6 +18,8 @@ cloud-init (23.4-0ubuntu1~22.04.1) jammy; urgency=medium
     - Revert behavior downstream, leave duplicate data
   * d/p/series: bring back retain-old-groups.patch.
     This patch was inadvertently dropped in 5d4a3cf.
+  * refresh patches:
+    - d/p/do-not-block-user-login.patch
   * Upstream snapshot based on 23.4. (LP: #2045582).
     List of changes from upstream can be found at
     https://raw.githubusercontent.com/canonical/cloud-init/23.4/ChangeLog
```
```